test/emphasis-fullPath.html

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
nitpick: It would be better if we rename this test case to `sankey-emphasis.html` and add a child case named `emphasis: 'trajectory'`. You can refer to `test/sankey.html`.